src/app/visual/toolbar/toolbar-visual.component.html

```diff
@@ -6,6 +6,20 @@
   id="screenshot-toolbar"
 >
   <sky-toolbar>
+    <sky-toolbar-item>
+      <button
+        class="sky-btn sky-btn-primary"
+      >
+        <sky-icon
+          icon="plus-circle"
+          [ngClass]="{
+            'sky-margin-inline-compact' : theme == 'modern'
```

Comment:
Why does your visual test need to wrap logic around themes? Shouldn't the theme switcher above take care of this for you?

Reply:
This is for a small spacing thing that can't be accomplished in-component due to css limitations until we create a button component. I can connect with you directly if you have more questions for this. So the theme switcher changes 99.9% of things but we have to add this class consumer side.

Reply:
Hmm.. maybe we should add an inline comment to this then? ...just to help remind ourselves for the visual tests.

Reply:
@Blackbaud-AlexKingman added the comment